oaknational/oak-ai-lesson-assistant
# feat: relevant lessons [AI-607]

## Description

- store `relevantLessons` on persisted chat
- I've opted not to store the data inside the lesson plan object as (1) it feels more like metadata than content, and (2) we don't want to be sending ids back and forth to the LLM
- we may want to consider where we store `basedOn` too, but that's for another time  

## How to test

- Create a lesson
- You should see that 'relevantLessons' is populated in `app_sessions.output`
